### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by Alondra87**
**Description**
In this project, I build a simple HTML list of To Do tasks, using webpack to bundle JavaScript and applying ES6 syntax.
_Activity:_[ [https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md#project-requirements](https://github.com/Alondra87/Awesome-book-ES6/pull/url)](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md)

**Applied requirements**

- [x] Set up a new project with webpack.
- [x] Create an index.html and an empty To Do List placeholder. 
- [x] Create an index.js file and set an array of some simple to do tasks. 
- [x] On page load render the dynamically created list of tasks in the dedicated placeholder. 
- [x] Create a style.css and set rules for the To Do List loaded by Webpack Style/CSS Loader.

### CODE REVIEW

File: src/modules/Tasks.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const taskses = [{ task: 'Double-tap to edit' }, { task: "Drag'n drop to reorder your list" }, { task: 'Manage all your lists in one place' }, { task: 'Resync to clear out the old' }];
```
Comment (m05e5) [inline src/modules/Tasks.js]: - [x] Please you are required to create a list of objects with description/task, completed, and index
image

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @Alondra87 , 

Good job so far. I really like what you did!
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [x] Please kindly make sure your list of todos is showing properly
**imp**
image

- [x] _**Please check the comments below**_

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alondra87** [comment]
Excuse me, I am not sure about how to show this well for you. Could you be more specific if I am losing some step please?
Because when I run `npm run start` it shows properly. I add a screenshot of this:
 
image


- [x] I created a list of objects with description/task, completed, and index.

File: src/modules/Tasks.js (hunk 1 above)
Comment (rloterh) [reply src/modules/Tasks.js]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Andrea,  

Your project is complete! 🥇 There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Highlights 👍
- Descriptive Pull Request
- App is running as intended
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alondra87** [comment]
- [x] To Do list: list structure - completed

**Pull request merged**